.travis.yml

```diff
@@ -80,9 +80,8 @@ before_install:
   - nvm use 4
   # Remove Ruby directive from Gemfile as this image has 2.2.5
   - sed -i '/^ruby/d' Gemfile
-  - mkdir -p ~/.apt-cache
   - if [[ "$UNSUPPORTED" != true ]]; then bash .ci/deps.sh; fi
-  - if [[ "$UNSUPPORTED" != true ]]; then bash .ci/deps.shellcheck.sh; fi
+  - if [[ "$UNSUPPORTED" != true ]]; then bash .ci/deps.cabal.sh; fi
```

Comment:
it makes no difference on travis, as its image doesnt have ghc's alex pre-installed.